firefox_puppeteer/ui/windows.py

```diff
@@ -42,11 +42,16 @@ def focused_chrome_window_handle(self):
         """
         with self.marionette.using_context('chrome'):
             return self.marionette.execute_script("""
-              Cu.import("resource://gre/modules/Services.jsm");
-              var win = Services.focus.activeWindow;
-              return win.QueryInterface(Ci.nsIInterfaceRequestor)
-                        .getInterface(Ci.nsIDOMWindowUtils)
-                        .outerWindowID.toString();
+              Components.utils.import("resource://gre/modules/Services.jsm");
+
+              let win = Services.focus.activeWindow;
+              if (win) {
+                return win.QueryInterface(Components.interfaces.nsIInterfaceRequestor)
+                          .getInterface(Components.interfaces.nsIDOMWindowUtils)
+                          .outerWindowID.toString();
+              }
+
+              return null;
```

Comment:
The test failure you are talking about here, I can only see at https://github.com/mozilla/firefox-ui-tests/blob/master/firefox_puppeteer/tests/test_windows.py#L150. For any other usage of this method I cannot see that it could fail. Which other instance are you talking about?